src/main/webapp/import.html

```diff
@@ -7,6 +7,33 @@
 <body>
     <div>
         <input name="input" type="text" id="input-text"/>
+        <select id="type">
```

Comment:
Is this particular list based on some recipes you've already gathered and want to import? Or did you just make it from memory?

Reply:
This list based on https://en.wikibooks.org/wiki/Category:Recipes_by_type list of types, excluding some repeated types or not so popular.